### ISSUE TITLE
Throwing shots vs shooting them with sling

### ISSUE THREAD

**Issue by igroglaz** [angband/angband#5340]
Strange: throwing a shot makes the same (or even more) dmg that shooting it with a sling. PWMA:
https://user-images.githubusercontent.com/39904162/164175774-79617578-2c09-440f-bcba-8dd45723d069.png

https://user-images.githubusercontent.com/39904162/164175845-609aa154-2f42-4c05-889a-896e386a4c02.png
I suppose in V it's the same

Logic: throwing stone with a sling should be much more effective than throwing. 
https://www.scientificamerican.com/article/whistling-sling-bullets-were-roman-troops-secret-weapon/#:~:text=In%20the%20hands%20of%20an,your%20head%2C%22%20Reid%20said.
**In the hands of an expert, a heavy sling bullet or stone could reach speeds of up to 100 mph (160 km/h):**

Source: PWMA ticket https://github.com/draconisPW/PWMAngband/issues/543

**Comment by backwardsEric** [comment]
From the point of view of realism, you're right, but the intent of the damage formula for thrown objects with the "THROWING" flag was, namely for the daggers, spears, and hammers, to make them useful.  Perhaps dial back the damage and give thrown objects with the "THROWING" flag an accuracy bonus?

**Comment by NickMcConnell** [comment]
Yes, I've struggled with this one.  I wanted THROWING on sling ammo because it felt right, and was thematic (Bilbo throwing stones at spiders), but maybe it's best just to drop it.

**Comment by backwardsEric** [comment]
Part of this is that thrown missiles that have the THROWING tag benefit from the player's state->to_d bonus (damage bonuses from strength and equipped items that are neither the melee weapon nor launcher) while other ranged attacks are unaffected by that bonus.  Should ranged damage from slings also benefit from state->to_d?  Like thrown weapons, the sling missile's energy is tied to the player's motion at the time of launch.  With crossbows and bows, the energy in the missile is from what was stored in the launcher.  Perhaps thrown (and if allowed sling) missiles do not use the full state->to_d for melee attacks but have an altered damage bonus calculation that reflects that the player can not put their body weight into the ranged attack - limiting strength's damage bonus by the missile's weight, for instance.

Pushing the idea farther with regards to slings, should there be an object flag (STORED_ENERGY?; or, instead of an object flag, a separate tval?) for some types of launchers (crossbows, bows, slingshots, firearms) so their missiles only benefit from damage bonuses on the launcher and missile while other types of launchers without that flag (slings, atlatls) can get damage bonuses from other sources as well?  How should the launcher multiplier be handled for the latter types of launchers?  Perhaps, like it currently acts for all launchers, apply it only to the base damage of the missile plus the bonuses from the launcher and missile.  Consider restricting how much an ego can boost the multiplier for those launchers (unlike a bow or crossbow, the only mundane way to boost the sling's power is to increase the length of the sling's cords).

In terms of gameplay, the above ideas are a mixed bag:

1) The THROWING tag was introduced to make thrown daggers, spears, ... a more feasible tactic.  Restricting how much of state->to_d can be applied to them interferes with that.

2) Does applying other damage bonuses to slings make them overpowered?
